server.js

```diff
@@ -25,6 +25,9 @@ var stats = {
     tunnels: 0
 };
 
+// holds the CLI options
+var options;
```

Comment:
First of all, it isn't a global, because it's Node. It's just file scope. You have [18 other variables](https://github.com/CWSpear/localtunnel-server/blob/subdomain/server.js#L1-L26) that have so-called "global scope." I'm not sure why this one is not okay all of a sudden? This really is perfectly acceptable in Node. (Think of it as if the file were being executed in an IIFE).

I don't necessarily like the `ENV` approach, since it seems like it matches the `--port` option and is consistent with how both the server and client work.